src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,125 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    private static class Node<T> {
+        private T value;
+        private Node<T> next;
+
```

Comment:
Task description says "create Node objects to store data that have links to the left and to the right neighbour Node".
you should add link to previous Node, too.